product_docs/docs/postgres_distributed_for_kubernetes/1/recovery.mdx

```diff
@@ -221,14 +221,14 @@ spec:
         maxParallel: 8
 ```
 
-For more information, please see [EDB Postgres for Kubernetes recovery from volumeSnapshot objects](/postgres_for_kubernetes/latest/recovery/#recovery-from-volumesnapshot-objects).
+For more information, see [Recovery from volumeSnapshot objects](/postgres_for_kubernetes/latest/recovery/#recovery-from-volumesnapshot-objects) in the EDB Postgres for Kubernetes documentation.
 
 ## PITR from volumeSnapshot
 
-Same as when doing recovery from an object store, you can instruct PostgreSQL to halt the replay of Write-Ahead Logs (WALs)
-at any specific moment during volumeSnapshot recovery.
+You can instruct PostgreSQL to halt the replay of write-ahead logs (WALs)
+at any specific moment during volumeSnapshot recovery. This is the same capability as when recovering from an object store.
```

Comment:
Please check this edit. The introductory clause was awkward. I think this is what was meant?

Reply:
agree with the edit.